### ISSUE TITLE
https://github.com/vimperator/vimperator-labs/pull/587 commit  causes lagging.

### ISSUE THREAD

**Issue by stepshal** [vimperator/vimperator-labs#612]
##### Issue type:

- Bug report

##### Version:

```
<!-- (Paste the output of `:version` here) -->
Vimperator:	3.15.0 (created: 2016/10/08 06:55:36)
Firefox:	Mozilla/5.0 (X11; Linux i686; rv:50.0) Gecko/20100101 Firefox/50.0
```

##### Description:

When i updgraded vimperator to version 3.15 i discovered constant lagging when i tried work with commandline. For example, when i tried to type "ab" and autocomplete to about:support, commandline and whole browser freezes and not let's me to autocomplete.

**Comment by SammysHP** [comment]
With a fast machine this is only slightly noticeable, but you _are_ able to notice it. On a slower machine I observed a rather long lag.

Maybe it's the regular expression matching that slows it down.

Until a solution is found I suggest to revert 52b7f94e46dd09057e30c9c62e2f3233d68c5eb2.

**Comment by maxauthority** [comment]
But it still seems to be here for @stepshal and probably others? If this commit is really the cause, I am for reverting it. Working on the commandline should always be fast.

**Comment by stepshal** [comment]
@maxauthority Yes, i am for reverting it. For some reason with current build first entry is autocompletes without lag, but if you try to autocomplete to the next, lag is still here.

**Comment by SammysHP** [comment]
@stepshal 
That is interesting. Can you please describe exactly what you do to reproduce this?

I think it's not the completion but the history that became slower. And I tried it on an old (=slow) computer and can reproduce it with this sequence:

1. Press <kbd>:</kbd><kbd>s</kbd><kbd>e</kbd><kbd>t</kbd>
1. Press <kbd>↑</kbd> multiple times, each results in a slight delay

**Comment by stepshal** [comment]
Maybe it is history. 
For me:
1. Press `:tabopen`
2. Type  <kbd>a</kbd><kbd>b</kbd>
3. Press  <kbd>↑</kbd> to autocomplete to `about:pages`, each result in a 1 second delay (3.15 version), and only after first entry delay appears (master).

**Comment by stepshal** [comment]
@timss maybe just revert this commit?
